dist/tools/uf2/nrf52_softdevice_check.sh

```diff
@@ -24,10 +24,17 @@ if [ -z "${MOUNTPOINT}" ]; then
     exit 1
 fi
 
+if [ ! $(echo "${MOUNTPOINT}" | wc -l) -eq 1 ]; then
```

Comment:
```suggestion
if [ ! "$(echo "${MOUNTPOINT}" | wc -l)" -eq 1 ]; then
```

To fix the Shellcheck warning.